flowback/ledger/views.py

```diff
@@ -0,0 +1,104 @@
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from rest_framework import status
+from flowback.ledger.selectors import account_list, transaction_list
+
+from flowback.ledger.services import account_create, account_update, account_delete, transaction_create, transaction_update, transaction_delete
+from flowback.common.pagination import LimitOffsetPagination, get_paginated_response
+from .serializers import AccountListInputSerializer, AccountListOutputSerializer, AccountSerializer, TransactionListOutputSerializer, TransactionSerializer
+
+
+class AccountListAPI(APIView):
+    class Pagination(LimitOffsetPagination):
+        default_limit = 20
+        max_limit = 100
+
+    def get(self, request):
+        serializer = AccountListInputSerializer(data=request.query_params)
```

Comment:
We call serializers that uses query_params a FilterSerializer (check code for more of these)